specs/altair/fork.md

```diff
@@ -38,7 +38,25 @@ Note that for the pure Altair networks, we don't apply `upgrade_to_altair` since
 
 ### Upgrading the state
 
-After `process_slots` of Phase 0 finishes, if `state.slot % SLOTS_PER_EPOCH == 0` and `compute_epoch_at_slot(state.slot) == ALTAIR_FORK_EPOCH`, an irregular state change is made to upgrade to Altair.
+`upgrade_to_altair` should run after processing the final epoch of Phase 0.
+The state transition logic is slightly modified near the fork boundary to catch
+this irregular case and handle the upgrade, even in the presence of skipped
+slots at the fork boundary. `process_slots_into_altair` is defined to demonstrate.
+After the upgrade has successfully occurred, the existing definition of `process_slots` will suffice.
```

Comment:
It will suffice, but it *must* be called, or gap slots right after the fork transition won't work.